### PR TITLE
Add explicit test cases for #134

### PR DESCRIPTION
**Pull request by ynsehoornenborg**
Added some explicit test cases for #134 
On latest master I was unable to reproduce the actual issue, that's why there is no update to the production code.

Consider #134 solved.

### CODE REVIEW

File: Philips.CodeAnalysis.Test/Maintainability/Maintainability/NoNestedStringFormatsAnalyzerTest.cs [hunk 1]
```diff
@@ -466,6 +466,52 @@ public void Test(out string errorMessage)
...
+class Foo
+{{
+	public void Err(FormattableString fs)
+	{{
+	}}
+
+	public void Test(out string errorMessage)
+	{{
+		errorMessage = this.ToString();
+		Logs.ServiceAudit.Err({0}{1});
```
Comment (bcollamore) [inline Philips.CodeAnalysis.Test/Maintainability/Maintainability/NoNestedStringFormatsAnalyzerTest.cs:486]: Remove "Logs.ServiceAudit."

File: Philips.CodeAnalysis.Test/Maintainability/Maintainability/NoNestedStringFormatsAnalyzerTest.cs [hunk 2]
```diff
@@ -466,6 +466,52 @@ public void Test(out string errorMessage)
...
+class Foo
+{{
+	public void Err(FormattableString fs)
+	{{
+	}}
+
+	public void Test()
+	{{
+		string errorMessage = ""Some text"";
+		Logs.ServiceAudit.Err(""{errorMessage}"");
```
Comment (bcollamore) [inline Philips.CodeAnalysis.Test/Maintainability/Maintainability/NoNestedStringFormatsAnalyzerTest.cs:508]: Remove "Logs.ServiceAudit."

**Review by bcollamore — CHANGES_REQUESTED** [review]

File: Philips.CodeAnalysis.Test/Maintainability/Maintainability/NoNestedStringFormatsAnalyzerTest.cs (hunk 1 above)
Comment (ynsehoornenborg) [reply Philips.CodeAnalysis.Test/Maintainability/Maintainability/NoNestedStringFormatsAnalyzerTest.cs]: Oops, removed

File: Philips.CodeAnalysis.Test/Maintainability/Maintainability/NoNestedStringFormatsAnalyzerTest.cs (hunk 2 above)
Comment (ynsehoornenborg) [reply Philips.CodeAnalysis.Test/Maintainability/Maintainability/NoNestedStringFormatsAnalyzerTest.cs]: Oops, removed

**Review by bcollamore — APPROVED** [review]

**Pull request merged**